components/Progress-Bar/progress-bar-initialization.js

```diff
@@ -0,0 +1,132 @@
+            lightThemeProgressBar.startFromOne = lightThemeStartFromOneElement.checked;
+            const baseNumber = document.querySelectorAll('#LightThemeBaseNumber')[0].value;
+            lightThemeProgressBar.init("LightThemeProgressBar", baseNumber, 0);
+        })
+    }       
+    
+    
+          
+
+    //Dark Theme scripts------------------        
+    //Initialize the progress bar with 12 steps.
+    const darkThemeProgressBar = Object.create(window.msdsProgressBar);          
+
+    const darkThemeProgressBarEl = document.getElementById("DarkThemeProgressBar");
+    if(darkThemeProgressBarEl) {
+        let darkThemeBaseNumberElement = document.querySelectorAll('#DarkThemeBaseNumber')[0]    
+        darkThemeBaseNumberElement.value = 12;
+        darkThemeProgressBar.init("DarkThemeProgressBar", darkThemeBaseNumberElement.value, 0); 
+
+        document.querySelector('#DarkThemePrevStep').addEventListener('click', event => {
+            darkThemeProgressBar.prev();
+        })
+    
+        document.querySelector('#DarkThemeNextStep').addEventListener('click', event => {
+            darkThemeProgressBar.next();
+        })
+    
+        document.querySelector('#DarkThemeBaseNumber').addEventListener('change', event => {
+            darkThemeProgressBar.init("DarkThemeProgressBar", event.target.value, 0);        
+        })    
+    
+        const darkThemeStartFromOneElement = document.querySelector('#DarkThemeStartFromOne');
+        darkThemeStartFromOneElement.addEventListener('change', event => {        
+            darkThemeProgressBar.startFromOne = darkThemeStartFromOneElement.checked;
+            const baseNumber = document.querySelectorAll('#DarkThemeBaseNumber')[0].value;
+            darkThemeProgressBar.init("DarkThemeProgressBar", baseNumber, 0);
+        })
+    }                  
+
+    //---
+
+    // progressBar = window.progressBar;
+    // let baseNumberElement = document.querySelectorAll('#BaseNumber')[0]
+    // if(baseNumberElement) {
+    //     let baseNumber = baseNumberElement.value;
+    //     progressBar.init(baseNumber);    
+
+    //     document.querySelector('.prevStep').addEventListener('click', event => {
+    //         progressBar.onclick = progressBar.prev();
+    //     })
+
+    //     document.querySelector('.nextStep').addEventListener('click', event => {
+    //         progressBar.next();
+    //     })
+
+    //     document.querySelector('#BaseNumber').addEventListener('change', event => {
+    //         progressBar.init(event.target.value);        
+    //     })    
+
+    //     const startFromOne = document.querySelector('#StartFromOne');
+    //     startFromOne.addEventListener('change', event => {        
+    //         progressBar.startFromOne = startFromOne.checked;
+    //         const baseNumber = document.querySelectorAll('#BaseNumber')[0].value;
+    //         progressBar.init(baseNumber);
+    //     })  
+    // }
```

Comment:
We should remove the commented code if we don't need it

Reply:
I just pulled this branch down to my local and its still here

Reply:
I don't know why but if you get the latest commit you should not have the commented line of codes